libcodechecker/libhandlers/check.py

```diff
@@ -490,7 +490,7 @@ def __update_if_key_exists(source, target, key):
         import traceback
         traceback.print_exc()
     finally:
-        if is_temp_output:
+        if is_temp_output or 'clean' in args:
```

Comment:
How is this change in harmony with line 464:

```python
if 'force' in args:
             setattr(analyze_args, 'clean', True)
```

Is there something else that needs to be cleaned and `analyze` does not clean by itself?

Reply:
Yes the cleanup should be set there.

Reply:
Done.